boltdesignsystem/bolt
# DS-243: Remove URL attribute on bolt-text

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-243](https://pegadigitalit.atlassian.net/browse/DS-243)

## Summary

Remove bolt-text URL since it is not used.

## Details

Remove URL and the tag alterations associated with it. Also remove old commented-out code.

## How to test

Check out the code I removed, and the demos.
Confirm this is not in use anywhere else. 

## Release notes

`url` prop has been removed from the Text component. To create text link going forward, pass Text Link element into the `text` prop.
